### PR TITLE
Sort update severity

### LINKED ISSUE #761: New Update dialog: strange order of severity options
"medium - urgent - high" order is quite unusual and raises question
whether it's intentional or not.

### PR DESCRIPTION
**Pull request by pcreech**
closes #761

### CODE REVIEW

File: bodhi/views/generic.py [hunk 1]
```diff
@@ -92,13 +92,14 @@ def work():
 @view_config(route_name='new_update', renderer='new_update.html')
 def new_update(request):
     """ Returns the new update form """
+    log.fatal(bodhi.models.UpdateSeverity)
```
Comment (lmacken) [inline bodhi/views/generic.py]: I'm guessing this log was left over from debugging?

**Comment by lmacken** [comment]
Thanks for the patch! Looks good to me other than that `log.fatal`.

**Comment by pcreech** [comment]
@lmacken Updated!

**Comment by lmacken** [comment]
Thanks! :+1:

**Pull request merged**